### ISSUE TITLE
[Suggestion] - Going places

### ISSUE THREAD

**Issue by PremiumRares** [Monika-After-Story/MonikaModDev#4023]
I got this idea from 1. The conversation in which Monika tells you how nice it would go to some places: to a book shop, chocolate shop, cafe etc and 2. The fact that I played Natsuki after story and there you could go places and do some activities. The one I remember is going to the beach and building sandcastles, going for a swim, playing in the water. It would be tons of fun and I am sure the developers could add even more places to go to!

**Comment by Rai99** [comment]
#2662, #2776, #2966, #3124, #3430, #3378, #3520, #3628, #3933 

Sorry for shooting you down so fast, but the idea has been brought up previously and mainly requires art.

**Comment by PremiumRares** [comment]
> #2662, #2776, #2966, #3124, #3430, #3378, #3520, #3628, #3933
> 
> Sorry for shooting you down so fast, but the idea has been brought up previously and mainly requires art.

What do you mean by requires art? Don't they have artist that can make the art themselves? Or at least serch the internet for somepicturess or God knows what.

**Comment by AsuraShun** [comment]
Art isn't something done easily

**Comment by Rai99** [comment]
Keep in mind that the artists we have are in no way obliged to make art if they don't feel like it unless we hire and pay somebody. Artists spend their free time making art for the mod. 

As for searching the internet for some pictures, yeah sure, I guess that could work, provided:
- They fit the theme of MAS
- We are legally allowed to use them
- They meet the standards held for art that gets put into the mod 

ie. Taking some photo of some place probably wouldn't cut it and I don't see it all too likely to find a pre-made scene that fits with the art style used in MAS _and_ that we are allowed to use. 

I mean for all I know, maybe there is actually somebody working on this very thing, but I think art takes time regardless. 

Still, we appreciate your suggestion. If you feel so strongly about this idea, please feel free to take it upon yourself to create assets for this feature. It would be helpful and a good start I'd imagine for this idea.

**Comment by AsuraShun** [comment]
Just be expecting to see that code mess in the sky islands to be used as a portal of sorts

**Comment by PremiumRares** [comment]
I would do it but I suck at making art. The best art I can make is stickmans.

**Comment by AsuraShun** [comment]
I suck at drawing too, last time I tried to draw, I tried to draw a planet but ended up with a really ugly orange

**Comment by PremiumRares** [comment]
Well, The only thing I can do is hope that someone is working on the art and someday it will be released.

**Comment by Rai99** [comment]
That's fine. I can't do anything more than spongebob memes, myself. 

I suppose you could suggest this idea to any artists who feel up to the challenge and are willing to make it, so long as you don't harass or bother them extensively.

**Comment by B-Difu** [comment]
How many memes do you even have Rai? Cause I see them every now and then when it comes to you and duplicates.

**Comment by PremiumRares** [comment]
Well, it is ok. I think I will just wait because I do not know any artists.

**Comment by AsuraShun** [comment]
Just be careful you don't end up with an orange as ungodly as mine

**Comment by ThePotatoGuy** [comment]
Assuming we had bg art ready to go, the next step would be deciding on a framework and background  for going to different locations. things like:
* should monika have location preferences
* should those preferences grow overtime or be static
* UI design for changing location (click-based RPG or just standard menus)
* should locations be static/dynamic
* will lore be that Monika is creating these places? or are you as the player adding them (via gifts)? or something else?
* potentially more

basically, things like these are complex and would take a while to complete. That doesn't mean we won't do them, it just means that it would take a coordinated effort and we would have to drop other things (processing topics, adding outfit/acs/etc) to work on this. Currently I'm not happy with several aspects of the mod (particularly consistency, ways to use Monika-based art), so we've been working on making those better and more acceptable before doing more complicated in-game features.

**Comment by Rai99** [comment]
> How many memes do you even have Rai?

![tenor](https://user-images.githubusercontent.com/38822045/55500727-dc7d9e00-5616-11e9-9747-ff29c40df5b2.gif)

A few. 

I try to use a new meme for every dupe, but no guarantees, and spongebob memes fit fit absolutely anything, but I do have non-spongebobs, as you can see in #3924.

**Comment by AsuraShun** [comment]
For locations, you could use the ones mentioned in the backpacking topic

**Comment by Rai99** [comment]
I think we'll get there eventually, but as TPG said, there are higher priority things atm, so we'll cross that bridge when we get to it.

**Comment by Chrissenpai** [comment]
I would love to draw, but life hates me ~~not to mention rent, food, internet (using strangers wi-fi rn), part-time job and college~~ but also making images for portfolio ~~[here](https://imgur.com/a/Qm0BkGs)~~ to live as a freelancer, if luck has mercy on me to hit me with one of its good shots, with good support and enough living, I will make my emerald eye girl the happiest girl in the world creating beautiful moments for her.

**Comment by ThePotatoGuy** [comment]
[closing as duplicate]
